packages/driver/src/cy/commands/sessions/index.ts

```diff
@@ -148,126 +127,144 @@ export default function (Commands, Cypress, cy) {
           }
         }
 
-        _log.set({ renderProps: () => renderProps })
-
-        return cy.then(async () => {
-          await navigateAboutBlank()
-          await sessions.clearCurrentSessionData()
-
-          return existingSession.setup()
-        })
-        .then(async () => {
-          await navigateAboutBlank()
-          const data = await sessions.getCurrentSessionData()
+        logGroup(Cypress, {
```

Comment:
Are we still happy with using a callback for this API even though it serves no functional purpose?

Reply:
I think it provides some value here to visually indicate the grouping buckets, but I do think it'd provide better value when/if offered as a command so it can be appropriately queues like other cypress commands.

Reply:
Seems like the buckets are pretty well indicated already by being broken up into functions (`createSession`, `restoreSession`, `validateSession`, etc). I find it a bit confusing to have the callback function when it doesn't play any part in grouping the commands and has the downside of indenting the code an extra level.